backend/src/database/migrations/2022-09-01-automations.ts

```diff
@@ -0,0 +1,226 @@
+import { QueryInterface } from 'sequelize/types'
+
+export const up = async (queryInterface: QueryInterface, sequelize) => {
+  const transaction = await queryInterface.sequelize.transaction()
+
+  try {
+    await queryInterface.createTable(
+      'automations',
+      {
+        id: {
+          allowNull: false,
+          primaryKey: true,
+          type: sequelize.UUID,
+        },
+        type: {
+          allowNull: false,
+          type: sequelize.STRING(80),
+          validate: {
+            notEmpty: true,
+          },
+        },
+        tenantId: {
+          allowNull: false,
+          type: sequelize.UUID,
+        },
+        trigger: {
+          allowNull: false,
+          type: sequelize.STRING(80),
+          validate: {
+            notEmpty: true,
+          },
+        },
+        settings: {
+          allowNull: false,
+          type: sequelize.JSONB,
+        },
+        state: {
+          allowNull: false,
+          type: sequelize.STRING(80),
+          validate: {
+            notEmpty: true,
+          },
+        },
+
+        // metadata
+        createdAt: {
+          allowNull: false,
+          type: sequelize.DATE,
+        },
+
+        createdById: {
+          allowNull: false,
+          type: sequelize.UUID,
+        },
+
+        updatedAt: {
+          allowNull: false,
+          type: sequelize.DATE,
+        },
+        updatedById: {
+          type: sequelize.UUID,
+        },
+
+        deletedAt: {
+          type: sequelize.DATE,
+          allowNull: true,
+        },
+        deletedById: {
+          type: sequelize.UUID,
+        },
```

Comment:
Currently yes - although I see that `deletedById` is not a thing and can be removed. Since I'm doing `AuditLog` as well for automations I'm guessing I can remove `paranoid` part and do actual deletes :).